doc/architecture/adr/0019-message-broker.md

```diff
@@ -0,0 +1,53 @@
+Message broker
+
+* Status:
+* Deciders: @anna-knudsen, @duizer, @ckr123, @Exobitt
+* Date: 2022-01-19
+
+---
+
+## Context and Problem Statement
+
+For our event driven architecture solution we need a message broker that can handle a large number of messages.
+We also want a message broker that is fairly easy to set up.
+
+---
+
+## Considered Options
+
+* Kafka
+* RabbitMQ
+* Redis
+* ActiveMQ
+
+---
+
+## Decision Outcome
+
+We are going with RabbitMQ
+
+## Rationale
+
+RabbitMQ seems to be a simple message broker and not much else, which is exactly what we need.
+It is a very popular, open source message broker with lots of support and documentation.
+It also seems very easy to set up.
+
+Kafka:
```

Comment:
The rendered page will not have a break line. So maybe:

* Make this bold
* Use a header (###)
* Delete the line